pkg/asset/installconfig/powervs/validation.go

```diff
@@ -49,6 +49,32 @@ func validateMachinePool(fldPath *field.Path, machinePool *types.MachinePool) fi
 	return allErrs
 }
 
+// ValidatePERAvailability ensures the target datacenter has PER enabled.
+func ValidatePERAvailability(client API, ic *types.InstallConfig) error {
+	capabilities, err := client.GetDatacenterCapabilities(context.TODO(), ic.PowerVS.Zone)
```

Comment:
here it says zone but the param name in the package definition says region, good have same name everywhere.

Reply:
weird! PER enablement is zone specific so I'm a bit surprised at that. For example, dal12 does not currently support PER but dal10 does!

Reply:
Yeah, the API says it's a region, but when I passed in `dal`, it didn't work, and I was told by the PowerVS team that I should pass in `dal10` (which is a zone), instead. So, there's nothing we can do on our end.